Module/randomize.py

```diff
@@ -316,6 +372,9 @@ def form_restriction(form_id,form_level):
         # grab everything that can't possibly be locked by items
         for i in plrpList:
             inventory += i.Items
+            # remove the starting inventory to prevent duplication
```

Comment:
Is this needed? In app.py there is a starting inventory backup taken, so not sure if you meant to fix this in both places.

Reply:
I found both were necessary. One of these fixes addresses what happens when the logic doesn't reject a seed, and the other fix addresses the case when logic does reject a seed.